CHANGELOG.md

```diff
@@ -8,7 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 ### Added
+
+## [0.2.3] - 2022-10-05
+### Added
 * Support for MySQL storage backend (#210) (by @MidasLamb)
+* Support for type restrictions in authorization models (#223)
+* Tuple validation against type restrictions in Write API (#232)
+* Upgraded the Postgres storage backend to use pgx v5 (#225)
+
+### Fixed
+* Close database connections after migration (#252)
+* Race condition in streaming ListObjects tests (#255)
```

Comment:
And also in the code itself :) https://github.com/openfga/openfga/pull/256

Reply:
Added. Thank you!